### PR TITLE
1.6

### PR DESCRIPTION
**Pull request by krapivaa**

### CODE REVIEW

File: src/TodoListItem.js [hunk 1]
```diff
@@ -9,13 +9,17 @@
 
 import React from 'react';
 
-export default function TodoListItem({title}) { //destructuring
+function TodoListItem({title, item, id, onRemoveTodo}) { //destructuring
```
Comment (ndowmon) [inline src/TodoListItem.js:12]:
```suggestion
function TodoListItem({key, title, onRemoveTodo}) { //destructuring
```

File: src/TodoListItem.js [hunk 2]
```diff
@@ -9,13 +9,17 @@
...
 
     
     return (
         <React.Fragment>
-            <li>{title}</li>
+            <li>
+                {title}
+                <button type="button" onClick = {() => onRemoveTodo(item.id)}>Remove</button>
```
Comment (ndowmon) [inline src/TodoListItem.js:19]:
```suggestion
                <button type="button" onClick = {() => onRemoveTodo(key)}>Remove</button>
```

File: src/TodoListItem.js (hunk 1 above)
Comment (krapivaa) [reply src/TodoListItem.js]: Thank you

File: src/AddTodoForm.js [hunk 3]
```diff
@@ -24,8 +25,9 @@ function AddTodoForm({onAddTodo}) {
     return(
         <div>
             <form onSubmit={handleAddToDo} >
-                <label htmlFor="todoTitle">Title </label>
-                <input id="todoTitle" name="title" type="text" value={todoTitle} onChange={handleTitleChange}></input>
+               <InputWithLabel todoTitle={todoTitle} handleTitleChange={handleTitleChange} label={"Title"}>
```
Comment (Iuliia-Shevtsova) [inline src/AddTodoForm.js:28]: You don't need to pass the `label `as a prop anymore because you use `children` property instead.

**Review by Iuliia-Shevtsova — COMMENTED** [review]
Very good work!!

**Pull request merged**